sympy/geometry/tests/test_line.py

```diff
@@ -192,7 +192,7 @@ def test_line_geom():
     assert r4.intersection(r7) == r7.intersection(r4) == [r7]
 
     # Segment contains
-    a, b = symbols('a,b')
+    a, b = symbols('a,b', real=True)
     s = Segment((0, a), (0, b))
     assert Point(0, (a + b)/2) in s
```

Comment:
I don't think we want this to fail when vanilla (non-real) symbols are used. I didn't catch the change that made it so -- is it possible to relax the test so we don't get Undecided?

```python
>>> var('a b')
(a, b)
>>> s = Segment((0, a), (0, b))
>>> Point(0, (a + b)/2) in s
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "sympy\geometry\line.py", line 914, in __contains__
    result = self.contains(other)
  File "sympy\geometry\line.py", line 1653, in contains
    raise Undecidable("Cannot determine if {} is in {}".format(other, self))
sympy.geometry.line.Undecidable: Cannot determine if Point2D(0, a/2 + b/2) is in
 Segment2D(Point2D(0, a), Point2D(0, b))
```